app/client/doc/CHANGELOG.md

```diff
@@ -71,3 +71,5 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Functionally mocked a keybase in the form of a json file (default: pk.json) that will contain the privatekey
 - CLI calling RPC via generated client
 - Default configuration handling/overrides
+
+<!-- GITHUB_WIKI: changelog/app -->
```

Comment:
```suggestion
<!-- GITHUB_WIKI: changelog/client -->
```

What happens in the case where multiple tags use the same path? It looks like this changelog and app/pocket/doc/CHANGELOG.md are doing just that.

Reply:
I'm adding the following TODO in `wiki_sync.py`:

```
# TECHDEBT: Need to verify and exit the CI with a non-zero code if there is a collision on `WIKI_TAG`
```

Reply:
Nvm, it was 3 lines of code:

```py
        if wiki_path_format in seen_wiki_paths:
            raise ValueError(f"Duplicate wiki path: {wiki_path_format}")
        seen_wiki_paths.add(wiki_path_format)
```